File: app/src/main/java/com/mahadum360/mahadum/auth/di/AuthActivityComponent.java
package com.mahadum360.mahadum.auth.di;

import com.mahadum360.mahadum.auth.AuthActivity;

import dagger.Subcomponent;

@AuthScope
@Subcomponent(modules = {AuthActivityContextModule.class})
public interface AuthActivityComponent {

    void inject(AuthActivity activity);
}

File: app/src/main/java/com/mahadum360/mahadum/auth/di/AuthActivityComponent.kt
package com.mahadum360.mahadum.auth.di

import com.mahadum360.mahadum.auth.AuthActivity

import dagger.Subcomponent

@AuthScope
@Subcomponent(modules = [AuthActivityContextModule::class])
interface AuthActivityComponent {

    fun inject(activity: AuthActivity)
}
